src/modules/client/mixins/helperMixin.js

```diff
@@ -121,18 +121,20 @@ export const helperMixin = {
         NotifyPositive('Email envoyé');
 
         await this.getUserHelpers();
-        this.openNewHelperModal = false;
+        this.resetAddHelperForm();
       } catch (e) {
         console.error(e);
-        if (e.data.statusCode === 409) return NotifyNegative('Cet email est déjà utilisé par un compte existant.');
         NotifyNegative('Erreur lors de la création de l\'aidant.');
       } finally {
         this.loading = false;
-        this.firstStep = true;
       }
     },
     async nextStep () {
       try {
+        this.loading = true;
+        this.$v.newHelper.local.email.$touch();
+        if (this.$v.newHelper.local.email.$error || !this.newHelper.local.email) return NotifyWarning('Champs invalides');
```

Comment:
tu n'as pas un $required sur l'email ? si oui, `!this.newHelper.local.email` est inutile non ?